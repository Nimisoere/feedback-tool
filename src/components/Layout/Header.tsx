import React from 'react'
import Link from 'next/link'

import { HEADER_NAV_ITEMS } from './layout-data/header-data';

const Header: React.FC = () => {
  return (
    <header
      className={
        `w-full sticky top-0 z-10 shadow bg-white sm:h-20 py-4 sm:py-0 h-auto flex items-center transition-all duration-500`
      }
    >
      <div className='container mx-auto flex flex-wrap justify-between'>
        <div>
          <Link href="/" className='font-bold text-2xl'>FEEDr.</Link>
        </div>
        <nav className='gap-4 sm:w-auto w-full flex justify-between'>
          {
            HEADER_NAV_ITEMS.map((nav) => (
              <Link className='btn-plain' key={nav.id} href={nav.url}>{nav.title}</Link>
            ))
          }
        </nav>
      </div>
    </header>
  )
}

export default Header