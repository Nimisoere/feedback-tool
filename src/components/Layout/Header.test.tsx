import React from "react";
import "@testing-library/jest-dom";
import { fireEvent, render, screen } from "@testing-library/react";
import Header from "./Header";

describe("Feedback Graph", () => {
  it("should match snapshot", async () => {
    const { getByText, container } = render(
      <Header />
    );
    expect(container).toMatchSnapshot();
  });
});
