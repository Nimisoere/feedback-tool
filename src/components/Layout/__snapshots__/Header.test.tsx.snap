// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feedback Graph should match snapshot 1`] = `
<div>
  <header
    class="w-full sticky top-0 z-10 shadow bg-white sm:h-20 py-4 sm:py-0 h-auto flex items-center transition-all duration-500"
  >
    <div
      class="container mx-auto flex flex-wrap justify-between"
    >
      <div>
        <a
          class="font-bold text-2xl"
          href="/"
        >
          FEEDr.
        </a>
      </div>
      <nav
        class="gap-4 sm:w-auto w-full flex justify-between"
      >
        <a
          class="btn-plain"
          href="/"
        >
          Give feedback
        </a>
        <a
          class="btn-plain"
          href="/view-feedback"
        >
          View feedback
        </a>
      </nav>
    </div>
  </header>
</div>
`;
